README.md

```diff
@@ -228,6 +228,7 @@ Format of pct-config.yml
 ---
 template:
   id: <a unique name>
+  author: <name/username/orgname/handle/etc>
```

Comment:
could we separate these with a pipe rather than a slash, to avoid confusion with 'author/template'
```suggestion
  author: <name|username|orgname|handle|etc>
```